### PR TITLE
Blocks: Add story for Canvas `className` prop, cleanup UI SB `preview.tsx

### PR DESCRIPTION
**Pull request by JReinhold**
This PR adds a story to showcase the `className` prop for the Canvas Block.

It also cleans up the `preview.tsx` for UI SB. The ´preview.tsx` was originally migrated from the official storybook, but since it no longer serves as a showcase for features a lot of unnused properties and configurations could be removed.

### CODE REVIEW

File: code/ui/.storybook/preview.tsx [hunk 1]
```diff
@@ -164,17 +172,16 @@ export const decorators = [
       case 'stacked': {
         return (
           <Fragment>
-            <Symbols icons={['folder', 'component', 'document', 'bookmarkhollow']} />
```
Comment (JReinhold) [inline code/ui/.storybook/preview.tsx:167]: I have no idea why or what this is supposed to do. It renders SVG elements, but they don't appear in the canvas (their height is 0). I want to remove it but I don't know if it's magically used somewhere else?

@ghengeveld git history shows you've worked with this, albeit 2 years ago so it's fair if you don't remember.

File: code/ui/blocks/src/blocks/Canvas.stories.tsx [hunk 2]
```diff
@@ -81,3 +83,41 @@ export const WithAdditionalActions: Story = {
...
+        code={dedent`<style>
+        .my-custom-classname {
+          background-color: #fd5c9355;
+        }
+      </style>
+      <Canvas className="my-custom-classname">
+        <StoryComponent of={BooleanStories.Undefined} />
+      </Canvas>`}
+      />
+      <style>
```
Comment (tmeasday) [inline code/ui/blocks/src/blocks/Canvas.stories.tsx:111]: This is an interesting thing you are doing here.

It seems like two things:

1. Documenting what the story is intended for as it's not self explanatory (fair enough). Would a code comment be enough here? Perhaps there should be a SB addon/feature that surfaces such story-level comments in the addons panel somehow.

2. Showing the actual source code of the story. Surely the source addon should be doing this for us? Or is there some reason that doesn't work in this case?

I also wonder if you'd prefer a MDX file for this documentation. It's all a bit mind boggling as we work on docs for docs in docs etc etc.

**Review by tmeasday — APPROVED** [review]
Cleanup is good with me.

File: code/ui/blocks/src/blocks/Canvas.stories.tsx (hunk 2 above)
Comment (JReinhold) [reply code/ui/blocks/src/blocks/Canvas.stories.tsx:111]: 1. You're describing my and Michael's vision for the `Description` block here, being able to describe any comments on stories, meta or components. Doesn't work here though, I think it only works in Docs.
2. I hadn't tried the `storysource` addon, but it worked. I've done that instead:
image


I'm not sure about the MDX file, I'd like there to be some consistency, but that might be unfeasable.

Comment (tmeasday) [reply code/ui/blocks/src/blocks/Canvas.stories.tsx]: > You're describing my and Michael's vision for the Description block here, being able to describe any comments on stories, meta or components. Doesn't work here though, I think it only works in Docs.

Ahh, so we need the description block to also function as an addon (similar to the args table and the source). This is something that we want to do, I think @shilman has a name he uses for that, something like "standalone blocks". 👍

Comment (JReinhold) [reply code/ui/blocks/src/blocks/Canvas.stories.tsx]: yeah he told me about a pretty great vision for it, where any addon could be a block, or vice versa.
Someday. To the moon. 🚀

Comment (tmeasday) [reply code/ui/blocks/src/blocks/Canvas.stories.tsx]: "Docs Unbundling" -- that's it.

**Comment by JReinhold** [comment]
A bunch of snapshots changed now because I removed the global `color = red` arg which causes all the icons to not be red anymore. I think this is good, however I added it back to the Icon stories so you can actually change the color of the icon if you want to.

File: code/ui/.storybook/preview.tsx (hunk 1 above)
Comment (JReinhold) [reply code/ui/.storybook/preview.tsx:167]: Turns out it is necessary for any stories that renders the sidebar with icons.
Added it back in via a graceful decorator for selected stories.

**Pull request merged**